python/ray/serve/_private/api.py

```diff
@@ -219,24 +210,18 @@ async def serve_start_async(
     controller, controller_name = (
         await ray.remote(_start_controller)
         .options(num_cpus=0)
-        .remote(detached, http_options, dedicated_cpu, grpc_options, **kwargs)
+        .remote(http_options, dedicated_cpu, grpc_options, **kwargs)
     )
 
     client = ServeControllerClient(
         controller,
         controller_name,
-        detached=detached,
     )
     _set_global_client(client)
-    logger.info(
-        f"Started{' detached ' if detached else ' '}Serve instance in "
-        f'namespace "{SERVE_NAMESPACE}".'
-    )
```

Comment:
I think we still such a log statement just remove the "detached" parts?

Reply:
Sounds good, I added the log statements back.